.ci/scripts/prepare_old_deps.sh

```diff
@@ -31,35 +31,6 @@ sed -i \
    -e '/systemd/d' \
    pyproject.toml
 
-# Use poetry to do the installation. This ensures that the versions are all mutually
-# compatible (as far the package metadata declares, anyway); pip's package resolver
-# is more lax.
```

Comment:
I think this was well-meaning but unnecessary. I also think `pip install`ing old deps is a better approximation of how the people who are using older deps are gonna be installing Synapse.